lib/contentstack/region.rb

```diff
@@ -2,5 +2,10 @@ module Contentstack
   class Region
     EU='eu'
     US='us'
+    AZURE_NA='azure-na'
+    AZURE_EU='azure-eu'
+    PROTOCOL='https://'
```

Comment:
@sunil-lakshman Please make different class for constant Region should no contain the constants other than DB regions.